Mage.Sets/src/mage/sets/fallenempires/InitiatesOfTheEbonHand.java

```diff
@@ -0,0 +1,139 @@
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.fallenempires;
+
+import java.util.UUID;
+import mage.MageInt;
+import static mage.Mana.BlackMana;
+import mage.abilities.Ability;
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.common.delayed.AtTheBeginOfNextEndStepDelayedTriggeredAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.BasicManaEffect;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ *
+ * @author MarcoMarin
+ */
+public class InitiatesOfTheEbonHand extends CardImpl {
+
+    public InitiatesOfTheEbonHand(UUID ownerId) {
+        super(ownerId, 16, "Initiates of the Ebon Hand", Rarity.COMMON, new CardType[]{CardType.CREATURE}, "{B}");
+        this.expansionSetCode = "FEM";
+        this.subtype.add("Cleric");
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // {1}: Add {B} to your mana pool. If this ability has been activated four or more times this turn, sacrifice Initiates of the Ebon Hand at the beginning of the next end step.
+        SimpleActivatedAbility ability = new SimpleActivatedAbility(Zone.BATTLEFIELD,
+                new BasicManaEffect(BlackMana(1)),
+                new ManaCostsImpl("{1}"));
+        ability.addEffect(new InitiatesOfTheEbonHandEffect());
+        this.addAbility(ability);  
+    }
+
+    public InitiatesOfTheEbonHand(final InitiatesOfTheEbonHand card) {
+        super(card);
+    }
+
+    @Override
+    public InitiatesOfTheEbonHand copy() {
+        return new InitiatesOfTheEbonHand(this);
+    }
+}
+
+class InitiatesOfTheEbonHandEffect extends OneShotEffect {
+
+    public InitiatesOfTheEbonHandEffect() {
+        super(Outcome.Damage);
+        this.staticText = "If this ability has been activated four or more times this turn, sacrifice {this} at the beginning of the next end step";
+    }
+
+    public InitiatesOfTheEbonHandEffect(final InitiatesOfTheEbonHandEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public InitiatesOfTheEbonHandEffect copy() {
+        return new InitiatesOfTheEbonHandEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Integer amount = (Integer) game.getState().getValue(source.getSourceId().toString() + "InitiatesOfTheEbonHand");
+        if (amount == null) {
+            amount = 0;
+            DelayedTriggeredAbility delayedAbility = new AtTheBeginOfNextEndStepDelayedTriggeredAbility(new InitiatesOfTheEbonHandResetEffect());
```

Comment:
Wow.. you added a new general ability? Cool! :)

Anyways, I'll revisit this after I get around to testing all the earlier sets with actual play (deeper testing), if it happens to not work.. I better move along 'cause I'm afraid of losing the motivation, this week I already slacked and did nothing =/ had to study C and gdb instead..